cli/cmd/sync.go

```diff
@@ -37,6 +37,8 @@ func NewCmdSync() *cobra.Command {
 
 func sync(cmd *cobra.Command, args []string) error {
 	ctx := cmd.Context()
+	log.Info().Msg("Start sync")
```

Comment:
I think we should move this to `syncConnection` and add `source` and `destinations=[]` to that as otherwise it's missing information and `sync_tmie`